public/locales/bg/auth.json

```diff
@@ -24,5 +24,30 @@
     "forgotten-password": {
       "instructions": "За да смените паролата си, моля въведете вашия email и ще ви изпратим потвърждение с инструкции."
     }
+  },
+  "profile": {
```

Comment:
This is not a proper file for the personal profile page as the `auth` namespace is used in the `login` and `register` pages